tests/cypress/integration/z-newfold-labs/wp-module-onboarding/2-intermediate-steps/0-get-started/TC_03-get-started-welcome.cy.js

```diff
@@ -1,17 +1,17 @@
 // <reference types="Cypress" />
-import { CheckDrawerDisabled } from '../wp-module-support/drawer.cy';
-import { CheckCardHeadingSubheading } from '../wp-module-support/header.cy';
+import { CheckDrawerDisabled } from '../../wp-module-support/drawer.cy';
+import { CheckCardHeadingSubheading } from '../../wp-module-support/header.cy';
 import {
 	CheckHelpPanelLinks,
 	CheckIllustrationPanel,
 	CheckInfoPanel,
 	CheckIntroPanel,
-} from '../wp-module-support/sidebar.cy';
+} from '../../wp-module-support/sidebar.cy';
 
 describe( 'Get Started Welcome Page', function () {
 	before( () => {
 		cy.visit(
-			'wp-admin/?page=nfd-onboarding&flow=ecommerce#/wp-setup/step/get-started/welcome'
+			'wp-admin/?page=nfd-onboarding#/wp-setup/step/get-started/welcome'
```

Comment:
Is the plan to not test the `commerce flow` for any of the steps where we have removed `&flow=ecommerce`?

Reply:
We are testing commerce flow but if we keep `&flow=ecommerce` in the url then we can't access those general onboarding pages as it will add WooCommerce plugin in the plugin install queue.

Reply:
Again we need some separation between `General` and `Commerce` tests.